SOURCES/0002-feat-drivers-add-CephFS-and-GlusterFS-drivers.patch

```diff
@@ -13,7 +13,7 @@ Subject: [PATCH 02/29] feat(drivers): add CephFS and GlusterFS drivers
  create mode 100644 drivers/GlusterFSSR.py
 
 diff --git a/Makefile b/Makefile
-index ad81be40..07504828 100755
+index ad81be4..0750482 100755
```

Comment:
Do you think your git command options, or git version, can be adapted so that the hashes remain on 8 characters and thus reduce the overall diff?

Reply:
:+1: